.ci/Jenkinsfile

```
@@ -43,7 +43,19 @@ pipeline {
           }
           axis {
             name 'PLATFORM'
-            values 'ubuntu-18 && immutable'
+            values 'ubuntu-18 && immutable', 'orka && darwin && x86_64'
```

`orka` is now deprecated, the supported labels are:

* `<labelString>darwin x86_64 bigsur macos11</labelString>`
* `<labelString>darwin x86_64 monterey macos12</labelString>`

I'd use `macos11` so it runs always in the same kind of workers, while using `darwin` it can use any of the above

```suggestion
            values 'ubuntu-18 && immutable', 'macos11 && x86_64'
```